langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -703,8 +705,49 @@ function testShiftOnTupleWithoutValuesForRestParameter() {
     error err = <error>res;
     var message = err.detail()["message"];
     string detailMessage = message is error ? message.toString() : message.toString();
-    assertValueEquality("{ballerina/lang.array}OperationNotSupported", err.message());
-    assertValueEquality("shift() not supported on type '[int,int...]'", detailMessage);
+    assertValueEquality("{ballerina/lang.array}SizeMismatch", err.message());
+    assertValueEquality("insufficient member size: expected at least '2', found '1'", detailMessage);
+}
+
+function testShiftOnTupleWithVariableInherentTypes() {
```

Comment:
Shall we add test cases that use union types like the following?
```ballerina
public function main() {
    [int|string, int|string, int|string...] tupleWithVariableInherentTypes1 = [1, 1, 1];
    var x = tupleWithVariableInherentTypes1.shift();
}
```

Reply:
Thank you for the suggestion. Added with 717906f159d143fc9b5c9c2fb35042521b60f2a0